components/raftstore/src/store/fsm/store.rs

```diff
@@ -393,7 +393,12 @@ where
     pub tick_batch: Vec<PeerTickBatch>,
     pub node_start_time: Option<TiInstant>,
     pub pending_latency_inspect: Vec<util::LatencyInspector>,
-    pub disk_usage: disk::DiskUsage,
+
+    /// Disk usage for the store itself.
+    pub self_disk_usage: DiskUsage,
+    /// Disk usage for other stores. The store itself is not included.
+    /// Only contains items which is not `DiskUsage::Normal`.
+    pub store_disk_usages: HashMap<u64, DiskUsage>,
```

Comment:
If a peer is removed by conf change, when does it remove from `store_disk_usages`?
It seems just doing it in `handle_reported_disk_usage` is not enough?

Reply:
Nice catch. The hash map is in `PollContext` and only contains `store_id` instead of `peer_id`, so the only problem is how to handle an offline store. I have no idea now. How about left a todo here?

Reply:
LGTM.